zookeeper-server/src/main/java/org/apache/zookeeper/Watcher.java

```diff
@@ -31,6 +31,8 @@
 @InterfaceAudience.Public
 public interface Watcher {
 
+    Watcher DUMMY_WATCHER = event -> { };
```

Comment:
Please do not add this constant to this public API.
If you want to do this change for tests let's create a class inside the test packages

Reply:
Make sense. I'd like to introduce a `DummyWatch` in test scope and corresponding updates.

Reply:
Updated